agent/engine/execcmd/manager_init_task_linux.go

```diff
@@ -198,11 +236,11 @@ func getExecAgentConfigHash(config string) string {
 	return base64.URLEncoding.EncodeToString(hash.Sum(nil))
 }
 
-var execAgentConfigFileExists = configFileExists
+var osStat = os.Stat
 
-func configFileExists(configFilePath string) bool {
-	if _, err := os.Stat(configFilePath); err == nil {
-		return true
+func fileExists(configFilePath string) bool {
```

Comment:
nit: I think it's easier to mock `fileExists` over `os.Stat`

Reply:
It is easier, buy by mocking the whole `fileExists` function, we also remove the bits of logic from test coverage... I think this is a good balance between complexity and coverage.... thoughts?

Reply:
I think we can also unit test the `fileExists` function if test coverage is a concern, but no strong opinion here, mocking `os.Stat` sounds fine too

Reply:
btw, the input argument is still called `configFilePath`, maybe this can `filePath` directly

Reply:
good catch! Changed :D